Renderer/OpenXRUtil.cpp

```diff
@@ -137,9 +137,9 @@ namespace Util {
                 return false;
             }
 
-            XrReferenceSpaceType referenceSpaces[referenceSpacesCount];
```

Comment:
Moving to C++ vectors from C Arrays is fine for me.  I initially wanted to keep them as C Arrays as this is meant as an instructional sample rather than a production code.  I think this is a fine change.